mlrun/serving/states.py

```diff
@@ -346,6 +346,53 @@ def to(
         parent._last_added = step
         return step
 
+    def set_flow(
+        self,
+        steps: list[Union[str, StepToDict, dict[str, Any]]],
+        force: bool = False,
+    ):
```

Comment:
Mention that you `:return:` the last step in the added flow.
Could even add an example where you're doing a `step1.set_flow(...).to()` or something similar.